tests/rx_antenna_pattern.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+
+import borealis_experiments.superdarn_common_fields as scf
+from utils.experiment_prototype import ExperimentPrototype
+from pydantic import ValidationError
+
+
+class RxAntennaPatternWrongDimsIntf(ExperimentPrototype):
```

Comment:
This test fails if the config file specifies 15 main and 3 intf connected antennas (i.e. one N200 disconnected)